clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/tags/EntityTagger.groovy

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.tags
+
+import com.netflix.spinnaker.clouddriver.model.EntityTags;
+
+interface EntityTagger {
+  public static final String ENTITY_TYPE_SERVER_GROUP = "servergroup"
+  public static final String ENTITY_TYPE_CLUSTER = "cluster"
+
+  void alert(String cloudProvider,
```

Comment:
The parameters are a bit out of control ... next time this is touched I'll introduce some sort of Builder to better encapsulate the different bits.